README.md

```diff
@@ -5,15 +5,17 @@
 
 #### Usage:
 
-    python3 dns_filter.py --verbose --port 10053 --to xyz.net --regexp "^abcd.+?.com$"
+    python3 dns_filter.py --verbose --port 10053 --cname xyz.net --ns1 work.offshore.ai. --ns2 auction.whois.ai. --regexp "^abcd.+?.com$"
```

Comment:
can you please change to : `python3 dns_filter.py --verbose --port 10053 --cname xyz.net --ns1 ns1.example.com. --ns2 ns2.example.com. --regexp "^abcd.+?.com$"` ?